features/install.feature

```diff
@@ -78,6 +78,29 @@ Feature: Installation
       And the file "config/initializers/localeapp.rb" should contain "config.api_key = ENV['LOCALEAPP_API_KEY']"
       And the exit status should be 0
 
+  Scenario: Running install with .env
+    In order to configure my non rails project and check my api key is correct
```

Comment:
Empty lines before and after added, description updated to describe the case's specific feature.